starboard/build/config/BUILD.gn

```diff
@@ -242,7 +243,12 @@ config("default_compiler_flags") {
 # TODO: b/329683364 - Move other flags setting the c++ standard from each
 # platform's platform_configuration to this config.
```

Comment:
It seems the decision was made to centralize the C++ standard selection switch in this file in #2484.

The TODO comment doesn't necessarily explain this very well, i'd expand it a bit. E.g 

```
This config is intended to encapsulate all platform-specific logic for selecting the C++ standard 
version in shared code. Starboard platform-specific build code should not attempt to overload this.
TODO:  Refactor the remaining related flags from each platforms platform_configuration to here.
```

Reply:
done